src/actions.js

```diff
@@ -167,22 +164,13 @@ export function fetchLocations(levels, type, parent) {
   return graphql(payload, `LOCATION_LOCATIONS_${type}`);
 }
 
-export function fetchLocationsStr(
-  mm,
-  level,
-  regions = null,
-  districts = null,
-  parent,
-  str,
-  first,
-) {
+export function fetchLocationsStr(mm, level, regions = null, districts = null, parent, str, first) {
   const types = mm.getConf("fe-location", "Location.types", ["R", "D", "W", "V"]);
   let filters = [`type: "${types[level]}"`, `str: "${str}"`, first && `first: '${first}'`].filter(Boolean);
   if (Boolean(parent)) {
     filters.push(`parent_Uuid: "${parent.uuid}"`);
-  }
-  else {
-    filter_location_by_parents(districts, regions, filters, types[level])
+  } else {
+    filter_location_by_parents(districts, regions, filters, types[level]);
```

Comment:
Please convert here the name of method to fulfil 'camelCase' if this is possible

Reply:
Done.